packages/store/testdata/issuer/source.js

```diff
@@ -0,0 +1,342 @@
+// Copyright (C) 2019 Agoric, under Apache License 2.0
+
+// @ts-check
+
+import { assert, details } from '@agoric/assert';
+import { makeExternalStore } from '@agoric/store';
+import { E } from '@agoric/eventual-send';
+import { Remotable } from '@agoric/marshal';
+import { isPromise } from '@agoric/promise-kit';
+
+import { makeAmountMath, MathKind } from './amountMath';
+import { makeInterface, ERTPKind } from './interfaces';
+
+import './types';
+
+/**
+ * @param {string} allegedName
+ * @param {AmountMathKind} [amountMathKind=MathKind.NAT]
+ * @returns {IssuerKit}
+ */
+function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT) {
```

Comment:
You seem to me making a copy of all of issuer.js, which will quickly get stale as issuer.js is improved. Wouldn't it be a better test all around to pare this down to only the relevant portions of issuer.js